controllers/grocerieController.js

```diff
@@ -60,20 +59,62 @@ const addItemToList = async (req, res) => {
   }
 };
 
+const checkedNotOnList = async (req, res) => {
+  try{
+    const groceries = await SavedList.find({checked: true}).populate('item');
+    console.log(groceries)
+    let deleted;
```

Comment:
- why not using const? 
- the name should be more precise (e.g. `deleteResponse`)